views/schedule_week.tmpl

```diff
@@ -63,12 +63,9 @@
             <tr>
               <td class="time-header mobile-only">{{.Hour | printf "%02d"}}:{{.Minute | printf "%02d"}}</td>
               {{if ne .RowSpan 0}}
-              <td rowspan="{{.RowSpan}}" class="schedule-timeslot schedule-blocks-{{.Item.Block}}" title="{{stripHTML .Item.Desc}}">
-                {{if .Item.PageURL}}
-                  <a href="{{url .Item.PageURL}}">{{.Item.Name}}</a>
```

Comment:
In removing the link href, if JS isnt loaded on the page, there will be no way to get to shows